src/portable/ReactUi/react/src/components/EditorPane.tsx

```diff
@@ -86,6 +88,7 @@ class EditorPane extends React.Component<IProps, typeof initialState> {
     private keyUp(event: any) {
         const { direction, setSaved, setInitialTranscription, lang, requestPosition, saved, selectedTask, totalSeconds, writeTranscription } = this.props;
         if (event.keyCode === 32) {
+            setSaved(true);
```

Comment:
We can leave this line out since when WriteTranscription is called, after it returns it sends a WRITE_FULFILLED action which will set saved to true. So saved is actually set to true after the data is really saved.